webgpu.h

```diff
@@ -721,6 +721,7 @@ typedef void (*WGPUProc)(void) WGPU_FUNCTION_ATTRIBUTE;
 
 typedef void (*WGPUDeviceLostCallback)(WGPUDeviceLostReason reason, char const * message, void * userdata) WGPU_FUNCTION_ATTRIBUTE;
 typedef void (*WGPUErrorCallback)(WGPUErrorType type, char const * message, void * userdata) WGPU_FUNCTION_ATTRIBUTE;
+typedef void (*WGPUUncapturedErrorCallbackInfo)(WGPUErrorCallback callback, void * userdata) WGPU_FUNCTION_ATTRIBUTE;
```

Comment:
WGPUUncapturedErrorCallbackInfo should be a struct, not a callback:
https://source.chromium.org/chromium/chromium/src/+/main:out/mac-Debug/gen/third_party/dawn/webgpu-headers/webgpu.h;l=1787-1791;drc=e7597cf3fea2525614c4c8c6f8044220287389d1

This file should mostly match the "upstream" version of webgpu.h generated by Dawn (`dawn/webgpu-headers/webgpu.h`). I'm sure it's diverged in other ways currently, but you should try to make the two generators match their results. (We will eventually stop having two separate generators for the same file, but not yet.)

`WGPUDeviceLostCallbackInfo` is also already implemented there, and it would make sense to update both together.

Also note: we are eventually going to want the YAML to define these CallbackInfo structs at a higher level (like, instead of a "struct", a new category for callback infos), so that other language bindings can generate things differently. However, I don't think we need to do this now, as the generator should hopefully be able to generate what we need as a "struct" type.